### PR TITLE
Release v4.0.0

### PR DESCRIPTION
**Pull request by ebfull**
This release activates [Canopy](https://z.cash/upgrade/canopy/) on the 4 year anniversary of the launch of the Zcash network!

### CODE REVIEW

**Review by str4d — APPROVED** [review]
utACK. I calculated the activation height myself, as well as checking it against ZIP 214.

**Review by daira — APPROVED** [review]
utACK

File: doc/release-notes/release-notes-4.0.0.md [hunk 1]
```diff
@@ -0,0 +1,290 @@
...
+
+elbandi (3):
+      Allow configure params directory
+      Add paramsdir option for manpage
+      Throw error if -paramsdir not a valid directory
+
+fanquake (8):
+      depends: set OSX_MIN_VERSION to 10.10
+      doc: mention that macOS 10.10 is now required
+      scripted-diff: prefer MAC_OSX over __APPLE__
```
Comment (defuse) [inline doc/release-notes/release-notes-4.0.0.md:255]: These underscores need to be escaped otherwise it'll render as bold text.

**Comment by ebfull** [comment]
@zkbot r+

**Pull request merged**